### PR TITLE
[ZH] Help Centre, Installation, Installation macOS, IRC, Main Page, Welcome, and Registration

### PR DESCRIPTION
**Pull request by Drickle**
Updated:    zh/Help_Centre, zh/Installation, zh/Internet_Relay_Chat, zh/Main_Page, zh/Welcome, zh/Registration

New:        zh/Installation/macOS

PS: Updates of other pages will cost lots of work xd

WIP: Help_Centre/Account_Restrictions (Finished,@ PR #3286)
Currently I will try to keep the translations updated as I can

### CODE REVIEW

File: wiki/Help_Centre/zh.md [hunk 1]
```diff
@@ -634,10 +622,12 @@ osu! 会自动保存你的所有成绩在 osu! 安装目录的 `/Data/r` 文件
 
 所有 `.osz` 文件都在歌曲目录中后，在歌曲选择界面按下 `F5`，会刷新游戏的谱面缓存并将你的新歌加载到游戏中。
 
-----------------------------------------------------------------------------
+---
 
 ## 网站
 
+### MISSING_HEADING
```
Comment (MegaApplePi) [inline wiki/Help_Centre/zh.md:629]: Can you translate this heading? `Common issues`

File: wiki/Help_Centre/zh.md [hunk 2]
```diff
@@ -786,10 +773,12 @@ osu! 安装程序自带有 .NET 框架，为了以防万一，你可以[在这
 
 大部分洋葱网络（Tor）或者其他与它有关的网络无法连接游戏服务器。这是为了避免恶意使用。
 
-----------------------------------------------------------------------------
+---
 
 ## 谱面制作与编辑
 
+### MISSING_HEADING
```
Comment (MegaApplePi) [inline wiki/Help_Centre/zh.md:780]: ditto
Can you translate this heading? `Common issues`

File: wiki/Help_Centre/zh.md [hunk 3]
```diff
@@ -955,11 +943,12 @@ osu! 安装程序自带有 .NET 框架，为了以防万一，你可以[在这
 
 如果你在过程中有任何问题，请发邮件给 [store@ppy.sh](mailto:store@ppy.sh)，我们很乐意帮助你。
 
-
-----------------------------------------------------------------------------
+---
 
 ## 诊断问题
 
+### MISSING_HEADING
```
Comment (MegaApplePi) [inline wiki/Help_Centre/zh.md:950]: ditto
Can you translate this heading? `Common issues`

**Comment by TPGPL** [comment]
reopen after you update the translations and resolve conflicts, consider making a new branch from master

**Pull request closed without merging**